app/src/main/kotlin/cz/covid19cz/erouska/db/SharedPrefsRepository.kt

```diff
@@ -393,11 +409,76 @@ class SharedPrefsRepository @Inject constructor(@ApplicationContext c: Context)
         return prefs.edit().putBoolean(EFGS_INTRODUCED, value).apply()
     }
 
-    fun wasHowItWorksShown() : Boolean{
+    fun wasHowItWorksShown(): Boolean {
         return prefs.getBoolean(HOW_IT_WORKS_SHOWN, false)
     }
 
-    fun setHowItWorksShown(){
+    fun setHowItWorksShown() {
         prefs.edit().putBoolean(HOW_IT_WORKS_SHOWN, true).apply()
     }
+
+    fun setVerificationData(code: String, token: String) {
+        prefs.edit().putString(VALIDATION_CODE, code)
+            .putString(VALIDATION_TOKEN, token).apply()
+    }
+
+    fun getVerificationCode(): String? {
+        return prefs.getString(VALIDATION_CODE, null)
+    }
+
+    fun getVerificationToken(): String? {
+        return prefs.getString(VALIDATION_TOKEN, null)
```

Comment:
Proč se tohle ukládá do SharedPreferences? Nestačí to v paměti?

Reply:
Kdyby appku ukončil, tak už potenciálně validní token nepoužije a bude muset žádat o nový kód